src/hotspot/share/classfile/javaClasses.cpp

```diff
@@ -23,6 +23,7 @@
  */
 
 #include "precompiled.hpp"
+#include "jvm.h"
```

Comment:
Does this really need jvm.h or just jvm_constants.h and jvm_io.h?

Reply:
jvm.h is needed because this file uses the JVM_XXX() functions.